routers/web/repo/activity.go

```diff
@@ -19,6 +19,7 @@ const (
 
 // Activity render the page to show repository latest changes
 func Activity(ctx *context.Context) {
+	AddHeaderCtxData(ctx)
 	ctx.Data["Title"] = ctx.Tr("repo.activity")
```

Comment:
if ctx.Written() {
		return
}

Reply:
Sorry I'm not familiar around these parts. What is `ctx.Written()`? Does it mean we can not add to `ctx` any more because it's considered "finished"?

Reply:
the AddHeaderCtxData  func does return the actuall feed if it got detected that the client want to have it

and so data is already written -> those we should return

Reply:
`ctx.Written` means whether any response has already been sent to the user.
It doesn't make sense to return two responses.